elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -269,12 +269,12 @@ private void createIndexMappings() {
 		}
 	}
 
-	private String analyzerName(DocumentFieldMetadata fieldMetadata) {
+	private String analyzerName(Class<?> entityType, DocumentFieldMetadata fieldMetadata) {
 		AnalyzerReference analyzer = fieldMetadata.getAnalyzer();
 		if ( analyzer.is( RemoteAnalyzerReference.class ) ) {
 			return analyzer.unwrap( RemoteAnalyzerReference.class ).getName();
 		}
-		LOG.analyzerIsNotRemote( String.valueOf( analyzer ) );
+		LOG.analyzerIsNotRemote( entityType, fieldMetadata.getFieldName(), analyzer.toString() );
```

Comment:
if you simply delete the "toString", you can have the formatter invoke that if it's needed.

That will avoid creating the String when the log level is disabled.. although it's WARN in this case so it shouldn't matter much, just pointing out that a `toString()` looks suspicious